__test__/support/environment/TestContext.ts

```diff
@@ -294,10 +294,10 @@ export default class TestContext {
           kind: configIntegrationKind,
         },
         serviceWorker: {
-          path: undefined,
-          workerName: undefined,
-          registrationScope: undefined,
-          customizationEnabled: true,
+          customizationEnabled: false,
```

Comment:
Looks like `customizationEnabled` is pre-existing dead code, so we could just clean this up instead of changing the value, since it has no effect anywhere.